src/main/java/org/devcourse/voucher/domain/voucher/amount/FixedAmount.java

```diff
@@ -7,7 +7,7 @@ public FixedAmount(int amount) {
     }
 
     @Override
-    public boolean validate(int amount) {
-        return 0 <= amount;
+    public boolean inValid(int amount) {
+        return 0 > amount || amount > 100_000;
```

Comment:
이건 제 취향일수도 있는데 보통 상수와 변수를 비교할때는 변수를 왼쪽에 두는 것 같아요.
```suggestion
        return amount < 0 || amount > 100_000;
```